packages/fxa-content-server/tests/intern.js

```diff
@@ -174,6 +174,12 @@ if (args.firefoxBinary) {
 
 const failed = [];
 
+intern.on('suiteEnd', (test) => {
```

Comment:
why do we need both 'suiteEnd' and 'testEnd'?

Reply:
Yea, unfortunatley `testEnd` event doesn't get triggered if a failure in the `beforeEach`.